students/Jaeyou-Jeong/westagram/users/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+class UserInfo(models.Model):
```

Comment:
User라는 class 명만으로도 충분히 UserInfor을 내포 합니다.
```suggestion
class User(models.Model):
```

Reply:
네 수정했습니다.